scripts/run_coverage.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -x
+
+# shellcheck source=/dev/null
+source "$HOME"/.cargo/env
+source "$(dirname "$0")"/test-util.sh
+
+PROJECT_DIR="/cloud-hypervisor"
+TARGET_DIR="$PROJECT_DIR/target"
+
+pushd $PROJECT_DIR || exit
+
+export BUILD_TARGET=${BUILD_TARGET-$(uname -m)-unknown-linux-gnu}
+
+# GLIBC > 2.31
+GRCOV_RELEASE_URL="https://github.com/mozilla/grcov/releases/download/v0.8.19/grcov-$BUILD_TARGET.tar.bz2"
+wget --quiet "$GRCOV_RELEASE_URL" || exit 1
+tar -xjf "grcov-$BUILD_TARGET.tar.bz2"
+
+rustup component add llvm-tools-preview
```

Comment:
Again is this something we could do in the container instead of the script?

Reply:
ditto.